packages/sampler/.storybook/preview.js

```diff
@@ -24,4 +50,11 @@ export const parameters = {
 		}
 	}
 };
+export const globalTypes = {
+	'debug aria': getBooleanType('debug aria'),
+	'debug layout': getBooleanType('debug layout'),
+	'debug spotlight': getBooleanType('debug spotlight'),
+	'locale': getObjectType('locale', 'en-US', locales),
+	'background': getObjectType('background', 'default', backgrounds)
```

Comment:
Would you please put these first?

Reply:
It's fixed. Now `locale` and `background` are shown first.